genbadge/utils_flake8.py

```diff
@@ -115,7 +117,14 @@ def parse_flake8_stats(stats_txt  # type: str
     for line in stats_txt.splitlines():
         match = RE_TO_MATCH.match(line)
         if not match:
-            warn("Line in Flake8 statistics report does not match template and will be ignored: %r" % line)
+            smatch = SOURCE_MATCH.match(line)
```

Comment:
Could you please implement the suggestion I made in https://github.com/smarie/python-genbadge/pull/21/files#r751296939 and https://github.com/smarie/python-genbadge/pull/21/files#r751299039 ?

 - putting the "if match" branch first
 - adding a comment in each branch to explain and give an example of what is handled in that branch

This is to help future maintainers and editors :)
Thanks !